public/src/components/campaign/sidebar_list_view/current_campaign.jsx

```diff
@@ -17,6 +12,7 @@ const CurrentCampaign = (props) => {
   }
   const handleClick = () => {
     const { id: campaignId } = id;
+    setVolunteerActive();
```

Comment:
Isn't this only set when the volunteer starts calling by clicking Start Calling on the calls page? Why are we triggering this when they're just going to the calls page?

Reply:
The start calling button is on this page, then this takes the volunteer to the control view (after setting connection to active) and the call pre-call view (in component did mount) assigns a call to the user.